ale/base/data_naif.py

```diff
@@ -435,47 +443,74 @@ def sensor_position(self):
-                if self.correct_lt_to_surface and self.light_time_correction.upper() == 'LT+S':
-                    obs_tar = pyspiceql.getTargetState(time,
-                                                       target,
-                                                       observer,
-                                                       'J2000',
-                                                       self.light_time_correction)
-                    obs_tar_lt = obs_tar.lt
-
-                    # ssb to spacecraft
-                    ssb_obs = pyspiceql.getTargetState(time,
-                                                       target,
-                                                       'SSB',
-                                                       'J2000',
-                                                       "NONE")
-                    ssb_obs_state = np.array(list(ssb_obs.starg))
-
-                    radius_lt = (self.target_body_radii[2] + self.target_body_radii[0]) / 2 / (scipy.constants.c/1000.0)
-                    adjusted_time = time - obs_tar_lt + radius_lt
-
-                    ssb_tar = pyspiceql.getTargetState(adjusted_time,
-                                                       target,
-                                                       'SSB',
-                                                       'J2000',
-                                                       "NONE")
-                    ssb_tar_state = np.array(list(ssb_tar.starg))
-                    state = ssb_tar_state - ssb_obs_state
-
-                    matrix = spice.sxform("J2000", self.reference_frame, time)
+            # spkezr returns a vector from the observer's location to the aberration-corrected
+            # location of the target. For more information, see:
+            # https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/FORTRAN/spicelib/spkezr.html
+            if self.correct_lt_to_surface and self.light_time_correction.upper() == 'LT+S':
+                print("BANANA", ephem)
+                obs_tars = self.spiceql_call("getTargetStates",{"ets": str(ephem),
+                                                                "target": target,
+                                                                "observer": observer,
+                                                                "frame": "J2000",
+                                                                "abcorr": self.light_time_correction,
+                                                                "mission": self.spiceql_mission})
+                # obs_tar = pyspiceql.getTargetState(time,
+                #                                     target,
+                #                                     observer,
+                #                                     'J2000',
+                #                                     self.light_time_correction)
+                obs_tar_lts = np.array(obs_tars)[:,-1]
```

Comment:
Why are these (and below) commented out?

Reply:
I left these here to remind myself of what was being replaced, they could be removed